pkg/provider/provider.go

```diff
@@ -54,7 +54,14 @@ func Provider() *schema.Provider {
 				Optional:      true,
 				DefaultFunc:   schema.EnvDefaultFunc("SNOWFLAKE_PRIVATE_KEY_PATH", nil),
 				Sensitive:     true,
-				ConflictsWith: []string{"browser_auth", "password", "oauth_access_token"},
+				ConflictsWith: []string{"browser_auth", "password", "oauth_access_token", "private_key"},
```

Comment:
Please add `"private_key"` to `ConflictsWith` for all others that currently have `"private_keyPath"`.

Reply:
done. PTAL.